Makefile

```diff
@@ -172,6 +172,10 @@ build/mgmt-%: $(GO_FILES) $(MCL_FILES) | bindata lang funcgen
 	@echo "Building: $(PROGRAM), os/arch: $*, version: $(SVERSION)..."
 	@time env GOOS=${GOOS} GOARCH=${GOARCH} go build -i -ldflags=$(PKGNAME)="-X main.program=$(PROGRAM) -X main.version=$(SVERSION) ${LDFLAGS}" -o $@ $(BUILD_FLAGS)
 
+.PHONY: install
```

Comment:
Is it common to add a new .PHONY here instead of the main one at the top? IOW does this work as expected?

Reply:
Oh I forgot there was a huge PHONY block at the top. I can move this up there.
Afaik you can have as many or few PHONY targets as you like. I've always tended to group the PHONY just above each respective rule but I will update to follow the existing style here